testsuite/tests/apicast/policy/websocket/test_websocket_policy_app_id.py

```diff
@@ -7,7 +7,9 @@
 from testsuite import TESTED_VERSION  # noqa # pylint: disable=unused-import
 from testsuite.tests.apicast.policy.websocket.conftest import retry_sucessful, retry_failing
 
-pytestmark = [pytest.mark.skipif("TESTED_VERSION < Version('2.8')")]
+pytestmark = [
+        pytest.mark.sandbag,  # deployment sensitive
```

Comment:
I would like to see why it is deployment sensitive and what it actually needs to be deployed.